src/App.css

```diff
@@ -7,6 +7,14 @@
   scroll-behavior: smooth;
 }
 
+:root {
+  --primary-broken-white: #e1e5f2;
+  --primary-grey: #575757;
+  --primary-grey-OP64: #57575764;
+  --primary-turquoise: #257a8b;
+  --primary-white: #fff;
+}
+
```

Comment:
👌

Reply:
Hi Iga. Thanks for the comments, have a great week :)